### ISSUE TITLE
Site broken when ckan.root_path set

### ISSUE THREAD

**Issue by joaotonelli** [ckan/ckan-docker#127]
Hi,

I need to set the **ckan.root_path** in the ckan.ini for a project that I work on.
When I do so, the site breaks. It seems to be something related to webassets and template locations.

Repro steps:
- Just set it in the **.env** file:
    CKAN__ROOT_PATH=/myroot

Here follows some printscreens:

![image](https://github.com/ckan/ckan-docker/assets/47873770/45832115-0414-436b-acdd-f244a64992b0)

![image](https://github.com/ckan/ckan-docker/assets/47873770/110192eb-df27-4f67-a754-38dfdd6cd619)

**Comment by Chealer** [comment]
Hum, is that CKAN__ROOT_PATH variable documented somewhere?

**Comment by joaotonelli** [comment]
Absolutely. Here follows the CKAN official doc page: [https://docs.ckan.org/en/2.10/maintaining/configuration.html#ckan-root-path](https://docs.ckan.org/en/2.10/maintaining/configuration.html#ckan-root-path)

**Comment by joaotonelli** [comment]
Hi,

I have finally managed to fix this problem.
Actually, I had to configure the nginx container to redirect the requests accordingly (using the **/myroot/** prefix).

I don’t know whether it is the best solution, but it seems to be working fine now.
